src/main/java/com/github/tomakehurst/wiremock/jetty/JettyHttpServer.java

```diff
@@ -192,10 +194,21 @@ protected Handler[] extensionHandlers() {
 
   @Override
   public void start() {
-    try {
-      jettyServer.start();
-    } catch (Exception e) {
-      throw new RuntimeException(e);
+    int retryCount = 0;
+
+    while (retryCount < MAX_RETRIES) {
+      try {
+        jettyServer.start();
+        break;
+      } catch (IOException bindException) {
+        retryCount++;
+        if (retryCount >= MAX_RETRIES) {
+          throw new RuntimeException(
```

Comment:
Our coding rules build plugin is (rightly) complaining about throwing a generic exception here. Please can we change this to `FatalStartupException`.

Reply:
ok, done